src/loaders/libcuda.c

```diff
@@ -51,6 +52,15 @@ int load_libcuda(void) {
 
   #include "libcuda.fn"
 
+  v = ga_lib_version(lib, cuInit);
+  if (v == -1)
+    fprintf(stderr, "WARNING: could not determine cuda driver version.  Some versions return bad results, make sure your version is fine\n");
+
+  if (v > 373.06) {
+    fprintf(stderr, "ERROR: refusing to load cuda driver library because the version is blacklisted\n");
```

Comment:
Give more information so the user know how to fix the error. Can you add in the error msg a few driver version that are know to be good.

Or better, give the link to the issue. This will give them the up to date status. We don't want the user that see this error to ask us questions, they should be able to work around the problem without contacting us.